src/test/java/by/onliner/tests/PositiveTests.java

```diff
@@ -0,0 +1,44 @@
+package by.onliner.tests;
+
+import by.onliner.driver.DriverManager;
+import by.onliner.dto.Menu;
+import by.onliner.dto.ShoppingCartModalPopup;
+import by.onliner.page.CatalogPageItem;
+import by.onliner.page.MainPage;
+import by.onliner.page.ShoppingCartPage;
+import by.onliner.util.Assertions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class PositiveTests extends AbstractTest{
+
+    @Test(description = "Test for removing item from the shopping cart and checking the cart for emptiness")
+    public void removeItemFromShoppingCartTest() {
+        //given
+        MainPage mainPage = new MainPage();
+        Menu menu = new Menu();
+        CatalogPageItem catalogPageItem = new CatalogPageItem();
+        ShoppingCartModalPopup shoppingCartModalPopup = new ShoppingCartModalPopup();
+        ShoppingCartPage shoppingCartPage = new ShoppingCartPage();
+
+        mainPage.openPage();
+        menu.waitForPageOpened();
+        mainPage.openCatalogFirstItem();
+        catalogPageItem.waitForPageOpened();
+        catalogPageItem.addToCart();
+        shoppingCartModalPopup.closeShoppingCartPopup();
+        menu.waitForPageOpened();
+        menu.openShoppingCartPage();
+
+        //when
+        shoppingCartPage.waitForPageOpened();
+        shoppingCartPage.removeItemFromCart();
+        String removedItemMessage = shoppingCartPage.getRemovedItemInformation();
+        DriverManager.getDriver().navigate().refresh();
```

Comment:
Please use refresh inside POs where it's needed. Make test logic clear for a user, your test steps should represent business logic of the app

Reply:
Fixed